mlir/lib/Dialect/Tosa/Transforms/TosaFolders.cpp

```diff
@@ -299,7 +299,7 @@ DenseElementsAttr transpose(DenseElementsAttr attr, ShapedType inputType,
     return transposeTypeRaw<uint16_t>(attr, inputType, outputType, permValues);
   }
 
-  return transposeType<APFloat>(attr, inputType, outputType, permValues);
+  return nullptr;
```

Comment:
```suggestion
  return transposeType<APFloat>(attr, inputType, outputType, permValues);
```

Reply:
Let's not return null and keep our transpose as it was. If we need upstream's transpose at a later point, we'll need to reconcile the rest.